Buildings/Fluid/HeatExchangers/RadiantSlabs/SingleCircuitSlab.mo

```diff
@@ -226,6 +226,10 @@ user's guide</a> for more information.
 revisions="<html>
 <ul>
 <li>
+January 06, 2016, by Thierry S. Nouidui:<br/>
+Renamed parameter <code>nSta2</code> to <code>nSta</code>.
```

Comment:
Can you check why this model shows up in the pull request. There should only be models from `CDL`. Maybe you need to merge `issue609_cdl` again to your branch?